IDE/LINUX-SGX/README.md

```diff
@@ -12,6 +12,14 @@ To create the static library, simply call make:
 
 This will create a local static library, libwolfssl.sgx.static.lib.a, that can be linked with SGX enclaves to access wolfSSL APIs using SGX hardware.
 
-Limitations:
+### Customization:
+    To enable wolfssl debug, add CFLAGS=-DDEBUG_WOLFSSL.
+    To enable wolfssl benchmark tests with enclave, specify: HAVE_WOLFSSL_BENCHMARK at build
+    To enable wolfcrypt testsuite with enclave, specify: HAVE_WOLFSSL_TEST at build
+
+For example, to enable all three:
+`make -f sgx_t_static.mk CFLAGS=-DDEBUG_WOLFSSL HAVE_WOLFSSL_BENCHMARK=1 HAVE_WOLFSSL_TEST=1`
```

Comment:
Or in this case example export to set the environment variables. But copy and pasting the example "make" gives error  HAVE_WOLFSSL_TEST=1: No such file or directory.